packages/react-card/src/stories/CardScale.stories.tsx

```diff
@@ -0,0 +1,117 @@
+import * as React from 'react';
+
+import { Body, Caption } from '@fluentui/react-text';
+
+import { Button } from '@fluentui/react-button';
+import { MoreVerticalRegular } from '@fluentui/react-icons';
+import { Card, CardHeader } from '../index'; // codesandbox-dependency: @fluentui/react-card ^9.0.0-beta
+import { makeStyles, shorthands } from '@griffel/react';
+import { tokens } from '../../../react-theme/src/tokens';
+
+const ASSET_URL = 'https://raw.githubusercontent.com/microsoft/fluentui/master/packages/react-card';
+
+const avatarMauricioURL = ASSET_URL + '/assets/avatar_mauricio.svg';
+
+const SampleCardContent = () => (
+  <>
+    <CardHeader
+      image={<img src={avatarMauricioURL} alt="Face of a person" />}
+      header={
+        <Body>
+          <b>Mauricio August</b> + 7 others edited
+        </Body>
+      }
+      description={<Caption>Artificial Intelligence Deck</Caption>}
+      action={<Button appearance="transparent" icon={<MoreVerticalRegular fontSize={20} />} />}
+    />
+  </>
+);
+
+const useStyles = makeStyles({
+  grid: {
+    display: 'flex',
+    flexWrap: 'wrap',
+    ...shorthands.gap('24px'),
+  },
+  container: {
+    width: '500px',
+    height: '200px',
+    position: 'relative',
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'center',
+    ...shorthands.border('5px', 'solid', tokens.colorNeutralStrokeAccessible),
+  },
+});
+export const CardScale = () => {
+  const styles = useStyles();
+
+  return (
+    <div className={styles.grid}>
+      <div>
+        <h1>Default (fixed size)</h1>
+        <div className={styles.container}>
+          <Card style={{ width: '400px', height: '75px', justifyContent: 'center' }}>
```

Comment:
arent those inline styles obfuscating the actuall functionality ?

Reply:
Good point, will create those classes to avoid code copying mistakes.